### PR TITLE
[T4-Automation][SS][ceph-qe-script]: CEPH-10649

### PR DESCRIPTION
**Pull request by anrao19**
Tier-4 automation: combination of s3 and swift versioning:
log: http://magna002.ceph.redhat.com/ceph-qe-logs/anuchaithra/test_s3_and_swift_versioning.console.log
CEPH-10649 - Attempt enabling versioning on a bucket that is S3 versioned

### CODE REVIEW

**Review by mkasturi18 — COMMENTED** [review]
Looks good

**Review by ckulal — COMMENTED** [review]
LGTM

File: rgw/v2/tests/s3_swift/test_swift_basic_ops.py [hunk 1]
```diff
@@ -575,6 +585,83 @@ def test_exec(config, ssh_con):
...
+                    "args": [
+                        bucket_name,
+                        {"X-Versions-Location": container_name},
+                    ],
+                }
+            )
+            log.info(f"container {container}")
+            if container:
+                raise TestExecError(
+                    f"enable versioning on bucket {bucket_name} with swift user should fail"
```
Comment (hmaheswa) [inline rgw/v2/tests/s3_swift/test_swift_basic_ops.py:662]: @anrao19 , please change the conditional and exception message to failure scenario
"enable versioning on bucket {bucket_name} with swift user failed"

Comment (anrao19) [reply rgw/v2/tests/s3_swift/test_swift_basic_ops.py]: done @hmaheswa

**Review by hmaheswa — APPROVED** [review]

**Review by ckulal — APPROVED** [review]

**Pull request merged**